template-resources/src/main/resources/serviceinfo.st

```diff
@@ -6,6 +6,11 @@ $openbracket$
         "documentation" : "$widInfo.(k).documentation$",
         "module" : "$widInfo.(k).category$",
         "icon" : "$widInfo.(k).icon$",
+        $if(widInfo.(k).mavenDepends)$
+            $widInfo.(k).mavenDepends:{k1|
+                "gav" : "$widInfo.(k).mavenDepends.(k1).group$:$widInfo.(k).mavenDepends.(k1).artifact$:$widInfo.(k).mavenDepends.(k1).version$",
```

Comment:
This is inside a looping construct so it may generate multiple "gav" properties. Not sure it is what you want ?